shared/loadgenerator/loadgenerator.go

```diff
@@ -55,14 +57,18 @@ type GeneratorOptions struct {
 	// RequestTimeout is the maximum time waiting for a response.
 	RequestTimeout time.Duration
 	// QPS is the total QPS generated by all threads.
+	// If NumSteps is specified, for every step this value will be doubled.
```

Comment:
"If NumSteps is specified" seems redundant.  How about "It will be doubled every step."?

Reply:
added loadfactors. dont need this